packages/api-key-stamper/src/encoding.ts

```diff
@@ -0,0 +1,61 @@
+import { pointDecode } from "./tink/elliptic_curves";
+
+/**
+ * Code modified from https://github.com/github/webauthn-json/blob/e932b3585fa70b0bd5b5a4012ba7dbad7b0a0d0f/src/webauthn-json/base64url.ts#L23
+ */
+export function stringToBase64urlString(input: string): string {
```

Comment:
I think this is correct actually, because the whole encoding is "base64url".

If we returned base64-encoded URL strings, then `Base64UrlString` would be cool. But here we're talking about base64url-encoded strings! ⚠️

Reply:
ah! TIL!